.cirrus.yml

```diff
@@ -1,9 +1,27 @@
 freebsd_instance:
   image_family: freebsd-12-0
 test_task:
+  # see https://cirrus-ci.org/guide/tips-and-tricks/#custom-clone-command
+  # for Golang, we need to have a src/owner/project structure
+  clone_script: |
+    export CIRRUS_WORKING_DIR=$CIRRUS_WORKING_DIR/src/${CIRRUS_REPO_FULL_NAME}
```

Comment:
for a lot of this, are you sure you can't just use go get?

Reply:
I have no clue - I'll gladly try it out. :)